logging/bin/deploy_osd.sh

```diff
@@ -143,6 +170,7 @@ helm $helmDebug upgrade --install v4m-osd \
     --values logging/opensearch/osd_helm_values.yaml \
     --values "$wnpValuesFile" \
     --values "$nodeport_yaml" \
+    --values "$airgapValuesFile" \
```

Comment:
Don't we need/want to reference $AIRGAP_HELM_REPO below (on line 177)?   Might be good to split line 177 below so the Helm chart name sits on its own line.

Reply:
We do need to add the $AIRGAP_HELM_REPO ... I might've missed this one because it was kind of flushed in like that.  I moved it to its own line and added the $AIRGAP_HELM_REPO parameter to it.  It will show up in the next push.